.github/workflows/cloudflare-publish.yml

```diff
@@ -2,16 +2,37 @@ on:
   push:
     branches:
       - staging
+      - nf/publish-apps
 
-name: staging
+name: Publish apps to Cloudflare Pages
```

Comment:
this also publishes to npm, no? or is that what you mean by dedupe npm publish?

Reply:
Yeah, wanted to look a bit on how to multistage/parallelize the job while still using the same build output, but wanted to ensure the CF Pages publishing bit worked as intended first.